app/ai_backend/test_genesis_api.py

```
@@ -0,0 +1,744 @@
+import pytest
+import json
+import asyncio
+from unittest.mock import Mock, patch, AsyncMock, MagicMock
```

Ruff (0.11.9)

`unittest.mock.Mock` imported but unused

Remove unused import

(F401)